apps/services/user-notification/src/app/modules/notifications/notificationsWorker.service.ts

```diff
@@ -33,7 +37,36 @@ export class NotificationsWorkerService implements OnApplicationBootstrap {
     await this.worker.run<CreateHnippNotificationDto>(
       async (message, job): Promise<void> => {
         const messageId = job.id
-        this.logger.info('Message received by worker ... ...', { messageId })
+        this.logger.info('Message received by worker 1.1', { messageId })
+
+        const notification = { messageId, ...message }
+        const messageIdExists = await this.notificationModel.count({
+          where: { messageId },
+        })
+
+        if (messageIdExists > 0) {
+          // messageId exists do nothing
+          this.logger.info('notification already exists 1.1', {
+            messageId,
+          })
+        } else {
+          // messageId does not exist
+          // write to db
+          try {
+            const res = await this.notificationModel.create(notification as any)
```

Comment:
We should be casting the object to any. We should define the sequelize model correctly. See my comment on the model.